### PR TITLE
Emergency Trauma Unit Equipment

### PR DESCRIPTION
**Pull request by KittenColony**
## About the PR
Due to budget cuts in another sector, stations in the drift have been given ETU surplus equipment, a hybrid between corporate armour and practical equipment, all while offering protection!

_ETU Equipment does not provide protection. Insurance does not cover accidents that occur while wearing ETU surplus equipment. TTE:tm: is not responsible for any lasting damage caused by ETU equipment._

## Why / Balance
It looks cool and its something else to wear for paramedics or field medics

## Technical details
Several RSI files, A few new YML files and more namespace specific folders

## Media
image
imageimage
imageimage
imageimage
imageimage

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
:cl:
tweak: Boiler suit name and desc were flipped in code
add: Added new ETU outfit
add: ETU gear can be found in Paramedic lockers or fabricated in Uniform printers

### CODE REVIEW

**Comment by KittenColony** [comment]
technically ready, I still have to do the webbing and bags though

**Comment by dffdff2423** [comment]
Those look amazing!

**Comment by KittenColony** [comment]
current issue, medkits show up as ERROR when in belt, don't show the equipped sprites

**Comment by Nairodian** [comment]
I can't tell which is the front side and which is the back side of the helmet. It's very unreadable.

**Comment by KittenColony** [comment]
> I can't tell which is the front side and which is the back side of the helmet. It's very unreadable.

the front has the mask and cross visible, as well as the "bulb" on the flashlight, though I do agree it could be done better

**Comment by KittenColony** [comment]
fuck the medkit wearables, I'm making that its own fucking PR

**Comment by KittenColony** [comment]
!!! READY FOR REVIEW !!!

**Comment by KittenColony** [comment]
last minute changes because I noticed that reptilian snouts don't fit right

**Comment by KittenColony** [comment]
stiiill working on this, not quite ready to be merged yet

**Comment by KittenColony** [comment]
if that conflict strangles this i stg, otherwise this is good for review

File: Resources/Prototypes/_CD/Entities/Objects/Misc/garmentbag.yml [hunk 1]
```diff
@@ -17,3 +17,22 @@
       - id: PlushieLizard
     sound:
       path: /Audio/Effects/unwrap.ogg
+
+- type: entity
+  suffix: POINTMAN
```
Comment (Nairodian) [inline Resources/Prototypes/_CD/Entities/Objects/Misc/garmentbag.yml:22]: I don't think this should be in all caps, as that's usually reserved for "DON'T FUCKING MAP THIS" or "THIS IS A DEBUG OR ADMIN ONLY TOOL, DON'T TOUCH IT" But I'm not going to hold up the whole PR over it.

Comment (KittenColony) [reply Resources/Prototypes/_CD/Entities/Objects/Misc/garmentbag.yml]: force of habit, fixed now

**Comment by KittenColony** [comment]
ITS GREEEEEN

**Comment by KittenColony** [comment]
🎉

**Comment by KittenColony** [comment]
@LankLTE ready for review 👍

**Pull request merged**